WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModel.swift

```diff
@@ -699,6 +715,20 @@ private extension ProductFormViewModel {
     }
 }
 
+// MARK: Blaze-related
+//
+extension ProductFormViewModel {
+    func checkIfBlazeIntroViewIsNeeded() {
+        if let controller = blazeCampaignResultsController, controller.numberOfObjects == 0 {
+            shouldShowBlazeIntroView = true
+        }
+    }
```

Comment:
Copying the logic in `BlazeCampaignDashboardViewModel`

Reply:
Ah yeah this makes sense. Addressed in https://github.com/woocommerce/woocommerce-ios/pull/11011/commits/7cc08fde5ad0a04b3be82451b7473f1468d99409